content/en/platform/corda/4.9/enterprise/app-upgrade-notes-enterprise.md

```diff
@@ -19,17 +19,17 @@ weight: 20
 Corda Enterprise Edition 4.9 fixes a security vulnerability in the JPA notary. Before upgrading to Corda Enterprise Edition 4.9, read the guidance on [upgrading your notary service](../../../../../en/platform/corda/4.9/enterprise/notary/upgrading-the-ha-notary-service.md).
 {{< /warning >}}
 
-## Upgrading from Corda open source
+## Upgrading from Corda Community Edition
 
-Before upgrading to Corda Enterprise Edition 4.9, upgrade your CorDapp to Corda open source 4.9. See [upgrading CorDapps to newer platform versions](../../../../../en/platform/corda/4.9/enterprise/app-upgrade-notes.md) for detailed instructions.
+Before upgrading to Corda Enterprise Edition 4.9, upgrade your CorDapp to Corda Community Edition 4.9. See [upgrading CorDapps to newer platform versions](../../../../../en/platform/corda/4.9/enterprise/app-upgrade-notes.md) for detailed instructions.
```

Comment:
```suggestion
Before upgrading to Corda Enterprise Edition 4.9, upgrade your CorDapp to Corda Community Edition 4.9. See [Upgrading CorDapps to newer platform versions](../../../../../en/platform/corda/4.9/enterprise/app-upgrade-notes.md) for detailed instructions.
```
Here you're quoting the title of an article, I always capitalize article titles in similar cases.